src/pages/products/sgid/cadastre/plss-sections.astro

```diff
@@ -0,0 +1,138 @@
+---
+import Layout from '@layouts/DataPage.astro';
+
+import type { IMetadata, IPageMetadata } from '@models/products/sgid/types';
+import { ProductType, SgidCategory } from '@models/products/sgid/types';
+
+import HubDownloads from '@components/data/HubDownloads.astro';
+import Metadata from '@components/data/Metadata.astro';
+import UpdateHistory from '@components/data/UpdateHistory.astro';
+
+import CardWithPopularLink from '@components/page/CardWithPopularLink.astro';
+import CardWithSmallLink from '@components/page/CardWithSmallLink.astro';
+import GridForMoreResources from '@components/page/GridForMoreResources.astro';
+import GridForYouMightLike from '@components/page/GridForYouMightLike.astro';
+import Section from '@components/page/Section.astro';
+
+import CommonConversions from '@models/_partials/CommonConversions.astro';
+import PlssGlance from '@models/_partials/PlssGlance.astro';
+import PlssPointId from '@models/_partials/PlssPointId.astro';
+import PlssTownshipRange from '@models/_partials/PlssTownshipRange.astro';
+
+import { dataPages } from '@data/downloadMetadata';
+
+export const metadata: IMetadata = {
+  title: 'Utah PLSS Sections GCDB',
+  description: `PLSS Sections GCDB is the second level of a hierarchical breakdown of the Public Land Survey System Rectangular surveys.`,
+  stewards: ['UGRC', 'BLM', 'SITLA', 'Utah Counties', 'Premier Data Services'],
+  type: ProductType.POLYGON,
+  category: SgidCategory.CADASTRE,
+};
+
+const page: IPageMetadata = {
+  ...metadata,
+  application: 'https://plss.utah.gov/',
+  updateHistory: [
+    `December 15, 2020`,
+    `October 15, 2019`,
+    `January 15, 2019`,
+    `January 1, 2018`,
+    `November 1, 2016`,
+    `August 28, 2015`,
+  ],
+
+  hub: {
+    ...dataPages[metadata.title],
+  },
+};
+---
+
+<Layout title={page.title} subTitle={page.updateHistory[0]} category={page.category} actionUrl={page.application}>
+  <Metadata slot="metadata" {...page} />
+
+  <Fragment slot="summary">
+    <p>
+      Public Land Survey System Townships first divisions (Sections) are normally Public Land Survey System Tracts or
+      Public Land Survey System Sections.
```

Comment:
**polish**: this makes very little sense to me describing the data but I'll take your word for it with a suggestion to improve it in the future.